# HG changeset patch
# User alta88@fixall.com
# Date 1714558536 -7200
# Parent  9abf7ccd40fc2ca6def90e3f4b79f950a3da9c24
1871741 - Use "published" before "updated" for atom feeds date.

diff --git a/mailnews/extensions/newsblog/FeedParser.jsm b/mailnews/extensions/newsblog/FeedParser.jsm
--- a/mailnews/extensions/newsblog/FeedParser.jsm
+++ b/mailnews/extensions/newsblog/FeedParser.jsm
@@ -977,15 +977,15 @@ FeedParser.prototype = {
 
       tags = this.childrenByTagNameNS(
         itemNode,
         lazy.FeedUtils.ATOM_IETF_NS,
-        "updated"
+        "published"
       );
       if (!tags || !this.getNodeValue(tags[0])) {
         tags = this.childrenByTagNameNS(
           itemNode,
           lazy.FeedUtils.ATOM_IETF_NS,
-          "published"
+          "updated"
         );
       }
       if (!tags || !this.getNodeValue(tags[0])) {
         tags = this.childrenByTagNameNS(
